### PR TITLE
Doc template

### PR DESCRIPTION
**Pull request by tomlenth**
updated template per discussion with Preethi.  Add xeon deploy placeholder.

### CODE REVIEW

File: examples/ChatQnA/ChatQnA_Guide.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+Need to include the architecture with microservices. Like the ones Xigui/Chun made and explain in a para or 2 on the highlights of the arch including Gateway, UI, mega service, how models are deployed and how the microservices use the deployment service. The architecture can be laid out as general as possible, maybe  calling out “for e.g” on variable pieces. Will also be good to include a linw or 2 on what the overall use case is. For e.g. This chatqna is setup to assist in ansewering question on OPEA. The microservices are set up with RAG and llm pipeline to query on OPEA pdf documents 
+
+Microservice Outline and Diagram
+================================
+
+Deployment
+**********
+
++--------------------------------------------=-----------------------------------------+
```
Comment (dbkinder) [inline examples/ChatQnA/ChatQnA_Guide.rst:52]:
```suggestion
+--------------------------------------------------------------------------------------+
```

File: examples/ChatQnA/ChatQnA_Guide.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+|                                                                                      |
++============================================+=========================================+
+| XEON Scalable Processors                   |Gaudi Servers                            |
+|                                            |                                         |
++--------------------------------------------+-----------------------------------------+
+| NNIDIA GPUs                                | AI PC                                   |
+|                                            |                                         |
++--------------------------------------------+-----------------------------------------+
+
++--------------------------------------------=-----------------------------------------+
```
Comment (dbkinder) [inline examples/ChatQnA/ChatQnA_Guide.rst:63]:
```suggestion
+--------------------------------------------------------------------------------------+
```

File: examples/ChatQnA/ChatQnA_Guide.rst [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+|                                                                                      |
++============================================+=========================================+
+| Xeon & Gaudi with GMC                      |Xeon & Gaudi without GMC                 |
+|                                            |                                         |
++--------------------------------------------+-----------------------------------------+
+| Using Helm Charts                          |                                         |
+|                                            |                                         |
++--------------------------------------------+-----------------------------------------+
+
++--------------------------------------------=-----------------------------------------+
```
Comment (dbkinder) [inline examples/ChatQnA/ChatQnA_Guide.rst:74]:
```suggestion
+--------------------------------------------------------------------------------------+
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
Stray = in the top table line prevents the table from being recognized :(

**Review by dbkinder — APPROVED** [review]

**Review by hshen14 — APPROVED** [review]

**Pull request merged**